src/app/domains/domains.service.spec.ts

```diff
@@ -92,17 +154,52 @@ describe('DomainsService', () => {
 
     service.create(domainReq).subscribe(response => {
       expect(response).toEqual(domainReq)
-      done()
     })
   })
 
-  it('should delete the domain when a delete request is sent', (done: DoneFn) => {
+  it('should throw error on created domain details when a domain is created', () => {
+    const domainReq = {
+      profileName: 'testDomain',
+      domainSuffix: 'testDomain.com',
+      provisioningCert: 'domainCertLongText',
+      provisioningCertPassword: 'password',
+      provisioningCertStorageFormat: 'string'
+    }
+
+    const error = {
+      status: 404,
+      message: 'Not Found'
+    }
+
+    httpClientSpy.post.and.returnValue(throwError(error))
+
+    service.create(domainReq).subscribe(null, err => {
+      expect(error.status).toBe(err[0].status)
+    })
+  })
+
+  it('should delete the domain when a delete request is sent', () => {
     const domainName = 'testDomain'
 
     httpClientSpy.delete.and.returnValue(of({}))
 
     service.delete(domainName).subscribe(() => {
-      done()
+    })
+
+    expect(httpClientSpy.delete.calls.count()).toEqual(1)
+  })
+
+  it('should throw error when a delete request is sent', () => {
+    const domainName = 'testDomain'
+    const error = {
+      status: 404,
+      message: 'Not Found'
+    }
+
+    httpClientSpy.delete.and.returnValue(throwError(error))
+
+    service.delete(domainName).subscribe(null, err => {
```

Comment:
subscribe is showing as deprecated when using null, provide a function ()=>{} and i think that should fix it

Reply:
done